apps/client/src/features/viewers/studio/StudioClock.tsx

```diff
@@ -83,16 +85,28 @@ export default function StudioClock(props: StudioClockProps) {
       <NavigationMenu />
       <ViewParamsEditor paramFields={studioClockOptions} />
       <div className='clock-container'>
-        <div className={`studio-timer ${showSeconds ? 'studio-timer--with-seconds' : ''}`}>{clock}</div>
+        <div className='studio-timer-warp'>
```

Comment:
Question: what does timer-warp mean?

Reply:
the warp is used to create the "backlight" effect